### PR TITLE
PORTALS-2669 - Ecosystem component updates

### PR DESCRIPTION
**Pull request by nickgros**
PORTALS-2669

### CODE REVIEW

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss [hunk 1]
```diff
@@ -17,46 +17,46 @@
...
         font-size: 16px;
-        padding: 10px;
-        &.isSelected,
+        padding: 20px;
         &:hover {
-          color: Portal.$primary-action-color;
           background-color: color.change(
             Portal.$primary-action-color,
-            $alpha: 0.08
+            $alpha: 0.04
```
Comment (nickgros) [inline apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:35]: Design doesn't seem to call for a hover state, but removing it completely would make the component worse

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss [hunk 2]
```diff
@@ -17,46 +17,46 @@
     margin-bottom: 20px;
   }
   .control-container {
-    background: white;
     display: flex;
-    border: 1px solid #dcdcdc;
```
Comment (nickgros) [inline apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:22]: Border and background were useful for the cancer complexity portal, where the component is not shown on a white background. Think it still looks ok: 

image

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss [hunk 3]
```diff
@@ -17,46 +17,46 @@
...
+      border-right: 2px solid #dcdcdc;
+    }
     .content-container {
       overflow-y: auto;
       flex: 3;
       padding-left: 30px;
       padding-right: 20px;
     }
     .markdown {
+      h3,
```
Comment (nickgros) [inline apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:56]: Some portals (ELITE) are using h3, others use h4

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss [hunk 4]
```diff
@@ -17,46 +17,46 @@
     margin-bottom: 20px;
   }
   .control-container {
-    background: white;
     display: flex;
-    border: 1px solid #dcdcdc;
-    height: 450px;
+    height: 500px;
```
Comment (nickgros) [inline apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:21]: Design doesn't show overflow. Should overflow/height be removed?

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss [hunk 5]
```diff
@@ -55,8 +54,8 @@
     }
     .markdown {
       h4 {
```
Comment (nickgros) [inline apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:56]: Some portals (ELITE, ADKP) are using h3, others use h4 (Cancer Complexity)

Suggest portals use h4 in their markdown to get this right

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss (hunk 2 above)
Comment (jay-hodgson) [reply apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:22]: I agree

File: apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss (hunk 1 above)
Comment (jay-hodgson) [reply apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss:35]: I agree

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**